### ISSUE TITLE
No gyms nor pokestops

### ISSUE THREAD

**Issue by xyc1993** [Tr4sHCr4fT/fastmap#3]
So i run the script via followin command:
python fastmap.py -l '52.223634, 21.0103650' -r 1000

And I got no gyms nor pokestops. What is more, it seems that no cells were scanned:
2016-08-19 14:45:15,396 [   fastmap] [ INFO] API online! Scan starts in 5sec...
2016-08-19 14:45:20,403 [   fastmap] [ INFO] Scanned 0 cells; got 0 Gyms, 0 Pokestops, 0 Spawns

**Comment by tcmaps** [comment]
Hi, you need to also add -f when you run it the first time or want to add another area

**Comment by xyc1993** [comment]
Thanks, it... changed.
Now I get this:
2016-08-19 14:58:15,295 [   fastmap] [ INFO] API online! Scan starts in 5sec...
2016-08-19 14:58:20,302 [   fastmap] [ INFO] Scan 1 of 3...
2016-08-19 14:58:20,305 [   fastmap] [ERROR] <type 'exceptions.OSError'>
2016-08-19 14:58:30,316 [   fastmap] [ERROR] <type 'exceptions.OSError'>
2016-08-19 14:58:40,327 [   fastmap] [ERROR] <type 'exceptions.OSError'>
2016-08-19 14:58:50,336 [   fastmap] [ERROR] <type 'exceptions.OSError'>
2016-08-19 14:59:00,342 [   fastmap] [ERROR] <type 'exceptions.OSError'>
2016-08-19 14:59:10,354 [   fastmap] [ERROR] <type 'exceptions.OSError'>
2016-08-19 14:59:20,362 [   fastmap] [ERROR] <type 'exceptions.OSError'>

**Comment by tcmaps** [comment]
works on my side... 

`
C:\Users\TC\workspace\pogo>fastmap.py -l "52.223634, 21.0103650" -r 1000 -f  
2016-08-19 15:51:55,944 [   fastmap] [ INFO] API online! Scan starts in 5sec...   
2016-08-19 15:52:00,950 [   fastmap] [ INFO] Scan 1 of 3...   
2016-08-19 15:52:02,507 [   fastmap] [ INFO] UPSERTed 4 Gyms, 109 Pokestops, 440 Spawns. Sleeping...   
2016-08-19 15:52:12,509 [   fastmap] [ INFO] Scan 2 of 3...   
2016-08-19 15:52:12,848 [   fastmap] [ INFO] UPSERTed 4 Gyms, 112 Pokestops, 101 Spawns. Sleeping...   
2016-08-19 15:52:22,848 [   fastmap] [ INFO] Scan 3 of 3...   
2016-08-19 15:52:23,377 [   fastmap] [ INFO] UPSERTed 4 Gyms, 157 Pokestops, 83 Spawns. Sleeping...  
2016-08-19 15:52:33,378 [   fastmap] [ INFO] Scanned 3 cells; got 12 Gyms, 378 Pokestops, 624 Spawns  
`

what OS? is your account banned probably?

**Comment by xyc1993** [comment]
My OS is Ubuntu 14.04 LTS and I checked, my account is not banned.
I've seen that fastmap requires tejado's pgoapi so I've cloned pgoapi into the directory of fastmap, so that the folder with pgoapi is in fastmap folder. Is that what I was supposed to do?

Oh! And btw, can the app check gyms' status? Like prestige points, whether it belongs to some team or not, or what kind of pokemon is stationed there?
